### ISSUE TITLE
Is it possbile to tune in to a broadcast and not request camera/microphone?

### ISSUE THREAD

**Issue by grimace** [muaz-khan/RTCMultiConnection#943]
I'm looking to have a client that just views the broadcast video/audio but doesn't have to request access to the camera.  I feel like there is a way to do this but I'm just not sure how.

**Comment by ansarnisargill** [comment]
@grimace No, it is not possible with a webrtc framework. It works by connecting peer streams. Although in the Broadcast mode it only plays a single stream but it does need streams from all peers to stabilize the connection.
